tests/integration/README.md

```diff
@@ -432,36 +430,14 @@ If you need to add a new test suite, it can be added to the [job configuration](
 
 ## Environments
 
-When running tests, only one environment can be specified. Currently, the framework supports the following:
-
-### Native Environment (Default)
-
-The test binaries run on the native platform either in-memory, or as processes. This is the default, however you can
-also explicitly specify the native environment:
-
-```console
-$ go test ./... -istio.test.env native
-```
-
-Note: this may require you to [enable forwarding from Docker containers to the outside world](https://docs.docker.com/network/bridge/#enable-forwarding-from-docker-containers-to-the-outside-world):
-
-```bash
-sudo sysctl net.ipv4.conf.all.forwarding=1
-sudo iptables -P FORWARD ACCEPT
-# On some machines, an additional rule may be needed to allow traffic from all `br-...` docker bridge interfaces
-sudo iptables -A INPUT -i br-+ -j ACCEPT
-```
-
-### Kubernetes Environment
-
 The test binaries run in a Kubernetes cluster, but the test logic runs in the test binary. To specify the Kubernetes
```

Comment:
The sentence "To specify the Kubernetes environment" can be removed because the PR removes the "--istio.test.env kube" parameter in the line followed by this sentence.